thinc/tests/layers/test_tensorflow_wrapper.py

```diff
@@ -64,11 +64,12 @@ def model(tf_model):
 def test_tensorflow_wrapper_roundtrip_conversion():
     import tensorflow as tf
 
-    xp_tensor = numpy.zeros((2, 3), dtype="f")
+    ops = get_current_ops()
+    xp_tensor = ops.alloc2f(2, 3, zeros=True)
     tf_tensor = xp2tensorflow(xp_tensor)
     assert isinstance(tf_tensor, tf.Tensor)
-    new_xp_tensor = tensorflow2xp(tf_tensor)
-    assert numpy.array_equal(xp_tensor, new_xp_tensor)
+    new_xp_tensor = tensorflow2xp(tf_tensor, ops=ops)
```

Comment:
Does this work? We don't have `tensorflow2xp` with this argument in `v8.0.x`?

```
% rg "def tensorflow2xp"
thinc/util.py
386:def tensorflow2xp(tf_tensor: "tf.Tensor") -> ArrayXd:  # pragma: no cover
```

Reply:
I think you might be looking at the original commit that was cherry picked from `master`? The second commit modifies that line to explicitly cast the tensor.

Reply:
Whoops, yeah, I was accidentally looking at an individual commit and not _Files changed_.